Formula/hyper.rb

```diff
@@ -0,0 +1,23 @@
+class Hyper < Formula
+  desc "Hyper_(make VM run as container) client for https://www."
+  homepage "https://www.hyper.sh"
+  url "https://github.com/hyperhq/hypercli.git",
+    :tag => "v1.10",
+    :revision => "603e437099a0dc94943ee10219b35d57221262fa"
+
+  head "https://github.com/hyperhq/hypercli.git"
+
+  depends_on "go" => :build
+
+  def install
+    ENV["GOPATH"] = buildpath
+    mkdir_p "src/github.com/hyperhq"
+    ln_s buildpath, "src/github.com/hyperhq/hypercli"
+    system "./build.sh"
+    bin.install "hyper/hyper"
+  end
+
+  test do
+    system "#{bin}/hyper", "--help"
```

Comment:
The other commands need `access key` and `secret key` to auth with cloud, so only this `help` command works, but I think it is enough to verify the binary.